src/main/java/com/rocs/osd/controller/person/PersonController.java

```diff
@@ -0,0 +1,26 @@
+package com.rocs.osd.controller.person;
+
+import com.rocs.osd.domain.person.Person;
+import com.rocs.osd.service.person.PersonService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
```

Comment:
why do you need person controller when you have controllers for each inherited class of Person?